tests/unit/fem/test_inversetransform.cpp

```diff
@@ -125,7 +125,7 @@ TEST_CASE("InverseElementTransformation",
       REQUIRE( mesh_file.good() );
 
       const int npts = 100; // number of random points to test
-      const int min_found_pts = 94;
+      const int min_found_pts = 93;
```

Comment:
This isn't your fault, but I am really not a fan of this unit test.  This unit test looks like it will just pass or fail randomly.  I  think it would be far better to test on a fixed set of points that we know will converge with the method we have chosen.

Reply:
The test uses a fixed random seed, so the set of "random" points will be different only when the `rand()` implementation is different.